docs/background/kubernetes/gardener/hibernation.md

```diff
@@ -0,0 +1,61 @@
+---
+description: How hibernation of Gardener-based Kubernetes clusters works
+---
+
+# Putting Gardener clusters in hibernation
```

Comment:
This is not a how-to guide; it's a background explainer. Thus, the proper heading is simply this:

```suggestion
# Hibernation
```